cfme/networks/balancer.py

```diff
@@ -0,0 +1,104 @@
+from utils import version
+from utils.appliance.implementations.ui import navigator, CFMENavigateStep, navigate_to
+from cfme.web_ui import (
+    Region, Quadicon, toolbar as tb, paginator
+)
```

Comment:
**REQUIRED**
You shouldn't use old web ui elements in the new code. Use widgetastic please.